src/components/TopBar/TopBar.vue

```diff
@@ -21,6 +21,13 @@
 
 <template>
 	<div class="top-bar">
+		<button v-if="hasSelectedVideo && !isGrid"
+			class="top-bar__button primary"
+			@click="handlefollowSpeaker">
+			<span
+				class="icon-user-white" />
+			{{ followSpeakerLabel }}
```

Comment:
Since this will be very long depending on the translation, maybe only as a tooltip?

Reply:
Imo a hover tooltip is not enough for this. We could do a dismiss-able tooltip with an instruction to how to go back to autopilot. Similar to what we de for overflowing videos in the grid view

Reply:
Anyway let's open issues